Rakefile

```diff
@@ -8,3 +8,9 @@ Yast::Tasks.configuration do |conf|
   conf.install_locations["desktop/*"] = File.join(Packaging::Configuration::DESTDIR, "/usr/share/applications/")
   conf.install_locations["mime/*"] = File.join(Packaging::Configuration::DESTDIR, "/usr/share/mime/packages/")
 end
+
+# This is required, because `yast-travis-ruby` binary calls `rake test:unit`
```

Comment:
`rake -W` will show that "test:unit" is already defined in https://github.com/yast/yast-rake/blob/613cdb654d963a9acd8e35439ddaa1125857345c/lib/tasks/test_unit.rake#L54
The catch is that for historical reasons it expects the directory to be named `test`, not `spec`.

Until yast-rake resolves this it would be probably best to symlink `test` -> `spec` here and remove this task, then both `rspec` and `rake test:unit` will work.

Reply:
yast-rake is also where we declare `--format documentation` in a common place

Reply:
In that case I will change the name of the file from spec to test etc.

Reply:
hmm.. ok. I created a symlink instead, your solution was much simpler. About the "--format documentation" should I move it to the rake file instead?